iothub/device/src/Common/Exceptions/DeviceMaximumQueueDepthExceededException.cs

```diff
@@ -8,7 +8,8 @@
 namespace Microsoft.Azure.Devices.Client.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when an attempt to enqueue a message fails because the message queue for the device is already full.
+    /// This exception actually corresponds to IoTHubQuotaExceeded. For more information, see <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-troubleshoot-error-403002-iothubquotaexceeded"/>.
```

Comment:
How about instead od simply saying "For more information, see...", we say "For more information on what causes this error and the steps to fix it, see ..."?

Reply:
Sure